src/env.test.ts

```diff
@@ -23,8 +23,6 @@ describe('@src/env', () => {
         POOL: '',
       }
       const actual = env()
-      // Gotcha: If this fails you need a token in .env.development.local
-      expect(typeof actual.VITE_KITTYCAD_API_TOKEN).toBe('string')
```

Comment:
I don't think a unit test should be dependent on whether or not an environment variable is actually set. This test should pass regardless of the contents of `.env.development.local`.

Reply:
sounds good.